connect/build.gradle

```diff
@@ -45,6 +45,7 @@ dependencies {
     api 'com.squareup.okhttp3:logging-interceptor:3.11.0'
     api 'com.squareup.retrofit2:retrofit:2.4.0'
     api 'com.squareup.retrofit2:converter-gson:2.4.0'
+    implementation 'com.google.android.material:material:1.0.0'
```

Comment:
What are we using this dependency for? Also, is it on purpose that it's `implementation` and not `api`?